internal/tools/users.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package tools
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/asgardeo/go/pkg/user"
+	"github.com/asgardeo/mcp/internal/asgardeo"
+	"github.com/mark3labs/mcp-go/mcp"
+	"github.com/mark3labs/mcp-go/server"
+)
+
+func GetCreateTestUserTool() (mcp.Tool, server.ToolHandlerFunc) {
```

Comment:
Added this with `Test` term to convey the idea that this is intended only for test user creation